README.md

```diff
@@ -15,6 +15,8 @@ dp-dimension-importer
 | DB_POOL_SIZE               | 20                                   | The number of database connections to maintain in a pool
 | DIMENSIONS_EXTRACTED_TOPIC | dimensions-extracted                 | The topic to consume messages from to when dimensions are extracted
 | DIMENSIONS_INSERTED_TOPIC  | dimensions-inserted                  | The topic to write output messages when dimensions are inserted
+| EVENT_REPORTER_TOPIC       | event-reporter                       | The topic to write output messages when any errors occur during processing an instance
```

Comment:
Not sure how much of a standard / how consistent this topic name is already, but the topic names should reflect the event that it holds instead of the destination service. 'event-reported'?